### PR TITLE
Update ITEM.tsv

### PR DESCRIPTION
**Pull request by wlghksdl2**
ITEM_20150717_005526    공격 시 2% 확률로 +600% 피해 - 2% chance to deal 600% increased damage - Is this on basic attack? Need to know context. 005558 specifically says BASIC ATTACK but 005526 doesn't?
005526 는 공격이라고 적혀있는데, 그것은 무슨 공격인가요? 005558 은 자세히 기본 공격이라고 적혀있는데, 005526도 똑같은 기본공격인가요? Context 를 좀더 알아야되겠네요.
ITEM_20150717_005604 - Need edit / confirmation. 확인해주세요.
Flavor Text 는 생각보다 어려워서 더 자세히봐야될것같네요. 지금은 조그마한 스탯 버프 번역했읍니다.

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -5513,20 +5513,20 @@ ITEM_20150714_005512	Deodorizing Potion
 ITEM_20150714_005513	A strong potion used for removing odors. It has an antiseptic scent to it. 
 ITEM_20150714_005514	Tilla Monastery Scriptures 
 ITEM_20150714_005515	Scriptures that were scattered by rampaging monsters. Bring them to Mohaim. 
-ITEM_20150717_005516	
-ITEM_20150717_005517	
+ITEM_20150717_005516	The strong blade seems to show the way of the Sword.
```
Comment (emailboxu) [inline ITEM.tsv]: "It's as if the powerful blade shows its user the way of the sword."

File: ITEM.tsv [hunk 2]
```diff
@@ -5513,20 +5513,20 @@ ITEM_20150714_005512	Deodorizing Potion
...
+ITEM_20150717_005517	To slice all between you and the enemy is this blade's true strength.
```
Comment (emailboxu) [inline ITEM.tsv]: "The true power of this blade lies in its ability to cleave through everything between you and your enemy."

Sounds more natural.

File: ITEM.tsv [hunk 3]
```diff
@@ -5513,20 +5513,20 @@ ITEM_20150714_005512	Deodorizing Potion
...
 ITEM_20150717_005518	
-ITEM_20150717_005519	
-ITEM_20150717_005520	
+ITEM_20150717_005519	The souls of the swordsmen fighting to death with sword in hand is reflected in this blade.
```
Comment (emailboxu) [inline ITEM.tsv:5519]: "The fighting spirit of the swordsmen who fought fearlessly in the face of death is reflected in this blade."

I think...

File: ITEM.tsv [hunk 4]
```diff
@@ -5513,20 +5513,20 @@ ITEM_20150714_005512	Deodorizing Potion
...
+ITEM_20150717_005520	One can stab with the 폼멜로, but there's no need to try it on purpose.
```
Comment (emailboxu) [inline ITEM.tsv]: The pommel can be used to stab as well, but there's no need to put effort into doing so.

폼멜 = pommel

File: ITEM.tsv [hunk 5]
```diff
@@ -5513,20 +5513,20 @@ ITEM_20150714_005512	Deodorizing Potion
...
-ITEM_20150717_005524	
-ITEM_20150717_005525	
-ITEM_20150717_005526	
+ITEM_20150717_005524	- Bash Skill Level +10
+ITEM_20150717_005525	- Repair Fee increased by 100%
+ITEM_20150717_005526	- 2% chance to deal 600% increased damage
 ITEM_20150717_005527	
 ITEM_20150717_005528	
-ITEM_20150717_005529	
+ITEM_20150717_005529	- All Swordsmen Skill Level increased by +2
```
Comment (emailboxu) [inline ITEM.tsv]: Level -> Levels

File: ITEM.tsv [hunk 6]
```diff
@@ -5551,12 +5551,12 @@ ITEM_20150717_005550
 ITEM_20150717_005551	
 ITEM_20150717_005552	
 ITEM_20150717_005553	
-ITEM_20150717_005554	
-ITEM_20150717_005555	
+ITEM_20150717_005554	- All Cryomancer Skill Level increased by +1
```
Comment (emailboxu) [inline ITEM.tsv]: Level -> Levels

File: ITEM.tsv [hunk 7]
```diff
@@ -5567,11 +5567,11 @@ ITEM_20150717_005566
 ITEM_20150717_005567	
 ITEM_20150717_005568	
 ITEM_20150717_005569	
-ITEM_20150717_005570	
-ITEM_20150717_005571	
+ITEM_20150717_005570	- All Hunter Skill Level increased by +2
```
Comment (emailboxu) [inline ITEM.tsv]: Level -> Levels

File: ITEM.tsv [hunk 8]
```diff
@@ -5583,8 +5583,8 @@ ITEM_20150717_005582
 ITEM_20150717_005583	
 ITEM_20150717_005584	
 ITEM_20150717_005585	
-ITEM_20150717_005586	
-ITEM_20150717_005587	
+ITEM_20150717_005586	- All Quarrel Shooter Skill Level increased by +2
```
Comment (emailboxu) [inline ITEM.tsv]: Level -> Levels

File: ITEM.tsv [hunk 9]
```diff
@@ -5598,10 +5598,10 @@ ITEM_20150717_005597
 ITEM_20150717_005598	
 ITEM_20150717_005599	
 ITEM_20150717_005600	
-ITEM_20150717_005601	
+ITEM_20150717_005601	- All Monk Skill Level increased by +2
```
Comment (emailboxu) [inline ITEM.tsv]: Level -> Levels

File: ITEM.tsv [hunk 10]
```diff
@@ -5620,7 +5620,7 @@ ITEM_20150717_005619
 ITEM_20150717_005620	
 ITEM_20150717_005621	
 ITEM_20150717_005622	
-ITEM_20150717_005623	
+ITEM_20150717_005623	- All Hoplite Skill Level increased by +2
```
Comment (emailboxu) [inline ITEM.tsv]: Level -> Levels

File: ITEM.tsv (hunk 4 above)
Comment (wlghksdl2) [inline ITEM.tsv]: How's "The pommel can be used to stab as well, but there's no need to try it"

File: ITEM.tsv (hunk 1 above)
Comment (wlghksdl2) [inline ITEM.tsv]: Ya this is a good edit. Thanks

File: ITEM.tsv (hunk 3 above)
Comment (wlghksdl2) [inline ITEM.tsv:5519]: This flavor text is a little tricky I think. Going to give it another look at the original text in few hours and have another go.

File: ITEM.tsv (hunk 4 above)
Comment (emailboxu) [inline ITEM.tsv]: Yeah that's good too.

File: ITEM.tsv (hunk 2 above)
Comment (wlghksdl2) [inline ITEM.tsv]: "The true power of this blade is its ability to cleave everything between you and your enemy"

How's this?

Comment (emailboxu) [inline ITEM.tsv]: Seems good.

File: ITEM.tsv (hunk 1 above)
Comment (wlghksdl2) [inline ITEM.tsv]: Btw, how do I edit the pull request? Or is the admin. responsible for doing that?

Comment (emailboxu) [inline ITEM.tsv]: Top of your PR, there are 3 tabs, go to the "Files Changed" tab and you can edit from there.

**Comment by wlghksdl2** [comment]
Thanks mate

**Comment by imcgames** [comment]
Thank you :smile: 

(I can't see the comments that went on but seems like things were answered? )

**Comment by wlghksdl2** [comment]
No, the issue is still there. 5526 simply states 공격. Not sure whether 공격 refers to all types of attacks or basic attack only.

**Pull request merged**